dependencies/pom.xml

```diff
@@ -103,8 +103,9 @@
         <postgresql.version>42.3.3</postgresql.version>
         <h2.version>1.4.181</h2.version>
         <mariadb.version>2.7.2</mariadb.version>
+        <opengauss.version>5.0.0-og</opengauss.version>
         <!-- db connection pool -->
-        <druid.version>1.2.7</druid.version>
+        <druid.version>1.2.12</druid.version>
```

Comment:
why upgrade the druid version, Is there any compatibility issue between this database and Druid?